lib/ExpensiMark.js

```diff
@@ -49,6 +49,38 @@ export default class ExpensiMark {
                 replacement: '$1<code>$2</code>$3',
             },
 
+            /**
+             * Apply the hereMention first because the string @here is still a valid mention for the userMention regex.
+             * This ensures that the hereMention is always considered first, even if it is followed by a valid userMention.
+            */
+            {
+                name: 'hereMentions',
+                regex: /(^|\s)@here\b/g,
```

Comment:
When I was testing this out, I noticed that `@here` places a space inside the element tags, while the `@email@test.com` doesn't. Is it intentional or can we fix it so it's consistent?

![image](https://user-images.githubusercontent.com/1228807/234990754-f64c3dad-3173-4348-ad41-4eb4a9d31201.png)

Reply:
Ah good catch! that's definitely not how it should be, so let's fix that. And add a test for it.